chrysom/basicClient.go

```diff
@@ -111,13 +110,13 @@ type Items []model.Item
 // NewBasicClient creates a new BasicClient that can be used to
 // make requests to Argus.
 func NewBasicClient(config BasicClientConfig,
-	getLogger func(context.Context) log.Logger) (*BasicClient, error) {
+	getLogger sallust.GetLoggerFunc) (*BasicClient, error) {
 	err := validateBasicConfig(&config)
 	if err != nil {
 		return nil, err
 	}
 	if getLogger == nil {
-		getLogger = func(ctx context.Context) log.Logger {
+		getLogger = func(ctx context.Context) *zap.Logger {
```

Comment:
This is exactly what `sallust.Get` is for.  The logging library should never hand out a `nil` logger.  Just use the default logger, which `sallust.Get` returns when there isn't one in the context.

Reply:
Cool, I've been seeing this in our other servers as well. I'll make a note and update those as well